Index: app/src/main/java/ru/practicum/android/diploma/ui/vacancy/VacancyViewModel.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/ru/practicum/android/diploma/ui/vacancy/VacancyViewModel.kt b/app/src/main/java/ru/practicum/android/diploma/ui/vacancy/VacancyViewModel.kt
new file mode 100644
--- /dev/null	(date 1715339479390)
+++ b/app/src/main/java/ru/practicum/android/diploma/ui/vacancy/VacancyViewModel.kt	(date 1715339479390)
@@ -0,0 +1,4 @@
+package ru.practicum.android.diploma.ui.vacancy
+
+class VacancyViewModel {
+}
Index: app/src/main/java/ru/practicum/android/diploma/ui/vacancy/VacancyFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/ru/practicum/android/diploma/ui/vacancy/VacancyFragment.kt b/app/src/main/java/ru/practicum/android/diploma/ui/vacancy/VacancyFragment.kt
new file mode 100644
--- /dev/null	(date 1715340080008)
+++ b/app/src/main/java/ru/practicum/android/diploma/ui/vacancy/VacancyFragment.kt	(date 1715340080008)
@@ -0,0 +1,6 @@
+package ru.practicum.android.diploma.ui.vacancy
+
+import androidx.fragment.app.Fragment
+
+class VacancyFragment: Fragment() {
+}
Index: app/src/main/res/layout/fragment_vacancy.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/fragment_vacancy.xml b/app/src/main/res/layout/fragment_vacancy.xml
new file mode 100644
--- /dev/null	(date 1715339517784)
+++ b/app/src/main/res/layout/fragment_vacancy.xml	(date 1715339517784)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+</androidx.constraintlayout.widget.ConstraintLayout>
Index: app/src/main/res/layout/fragment_filter.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/fragment_filter.xml b/app/src/main/res/layout/fragment_filter.xml
new file mode 100644
--- /dev/null	(date 1715339517779)
+++ b/app/src/main/res/layout/fragment_filter.xml	(date 1715339517779)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+</androidx.constraintlayout.widget.ConstraintLayout>
Index: app/src/main/java/ru/practicum/android/diploma/ui/main/MainFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/ru/practicum/android/diploma/ui/main/MainFragment.kt b/app/src/main/java/ru/practicum/android/diploma/ui/main/MainFragment.kt
new file mode 100644
--- /dev/null	(date 1715338626099)
+++ b/app/src/main/java/ru/practicum/android/diploma/ui/main/MainFragment.kt	(date 1715338626099)
@@ -0,0 +1,6 @@
+package ru.practicum.android.diploma.ui.main
+
+import androidx.fragment.app.Fragment
+
+class MainFragment: Fragment() {
+}
Index: app/src/main/res/navigation/main_navigation_graph.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/navigation/main_navigation_graph.xml b/app/src/main/res/navigation/main_navigation_graph.xml
new file mode 100644
--- /dev/null	(date 1715339271081)
+++ b/app/src/main/res/navigation/main_navigation_graph.xml	(date 1715339271081)
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/main_navigation_graph"
+    app:startDestination="@id/favoritesFragment">
+
+    <fragment
+        android:id="@+id/favoritesFragment"
+        android:name="ru.practicum.android.diploma.ui.favorites.FavoritesFragment"
+        android:label="FavoritesFragment" />
+    <fragment
+        android:id="@+id/mainFragment"
+        android:name="ru.practicum.android.diploma.ui.main.MainFragment"
+        android:label="MainFragment" >
+        <action
+            android:id="@+id/action_mainFragment_to_filterFragment"
+            app:destination="@id/filterFragment" />
+    </fragment>
+    <fragment
+        android:id="@+id/teamFragment"
+        android:name="ru.practicum.android.diploma.ui.team.TeamFragment"
+        android:label="TeamFragment" />
+    <fragment
+        android:id="@+id/filterFragment"
+        android:name="ru.practicum.android.diploma.ui.filter.FilterFragment"
+        android:label="FilterFragment" />
+</navigation>
Index: app/src/main/java/ru/practicum/android/diploma/ui/favorites/FavoritesFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/ru/practicum/android/diploma/ui/favorites/FavoritesFragment.kt b/app/src/main/java/ru/practicum/android/diploma/ui/favorites/FavoritesFragment.kt
new file mode 100644
--- /dev/null	(date 1715338626093)
+++ b/app/src/main/java/ru/practicum/android/diploma/ui/favorites/FavoritesFragment.kt	(date 1715338626093)
@@ -0,0 +1,6 @@
+package ru.practicum.android.diploma.ui.favorites
+
+import androidx.fragment.app.Fragment
+
+class FavoritesFragment: Fragment() {
+}
Index: app/src/main/java/ru/practicum/android/diploma/ui/filter/FilterFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/ru/practicum/android/diploma/ui/filter/FilterFragment.kt b/app/src/main/java/ru/practicum/android/diploma/ui/filter/FilterFragment.kt
new file mode 100644
--- /dev/null	(date 1715338891083)
+++ b/app/src/main/java/ru/practicum/android/diploma/ui/filter/FilterFragment.kt	(date 1715338891083)
@@ -0,0 +1,6 @@
+package ru.practicum.android.diploma.ui.filter
+
+import androidx.fragment.app.Fragment
+
+class FilterFragment: Fragment() {
+}
Index: app/src/main/res/layout/fragment_team.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/fragment_team.xml b/app/src/main/res/layout/fragment_team.xml
new file mode 100644
--- /dev/null	(date 1715337136914)
+++ b/app/src/main/res/layout/fragment_team.xml	(date 1715337136914)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+</androidx.constraintlayout.widget.ConstraintLayout>
Index: app/src/main/res/layout/fragment_favorites.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/fragment_favorites.xml b/app/src/main/res/layout/fragment_favorites.xml
new file mode 100644
--- /dev/null	(date 1715337136905)
+++ b/app/src/main/res/layout/fragment_favorites.xml	(date 1715337136905)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+</androidx.constraintlayout.widget.ConstraintLayout>
Index: app/src/main/java/ru/practicum/android/diploma/ui/filter/FilterViewModel.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/ru/practicum/android/diploma/ui/filter/FilterViewModel.kt b/app/src/main/java/ru/practicum/android/diploma/ui/filter/FilterViewModel.kt
new file mode 100644
--- /dev/null	(date 1715338891068)
+++ b/app/src/main/java/ru/practicum/android/diploma/ui/filter/FilterViewModel.kt	(date 1715338891068)
@@ -0,0 +1,6 @@
+package ru.practicum.android.diploma.ui.filter
+
+import androidx.lifecycle.ViewModel
+
+class FilterViewModel: ViewModel() {
+}
Index: app/src/main/res/layout/fragment_main.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/fragment_main.xml b/app/src/main/res/layout/fragment_main.xml
new file mode 100644
--- /dev/null	(date 1715337136910)
+++ b/app/src/main/res/layout/fragment_main.xml	(date 1715337136910)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+</androidx.constraintlayout.widget.ConstraintLayout>
Index: app/src/main/java/ru/practicum/android/diploma/ui/team/TeamFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/ru/practicum/android/diploma/ui/team/TeamFragment.kt b/app/src/main/java/ru/practicum/android/diploma/ui/team/TeamFragment.kt
new file mode 100644
--- /dev/null	(date 1715338574948)
+++ b/app/src/main/java/ru/practicum/android/diploma/ui/team/TeamFragment.kt	(date 1715338574948)
@@ -0,0 +1,23 @@
+package ru.practicum.android.diploma.ui.team
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+
+class TeamFragment: Fragment() {
+//    private var _binding: FragmentTeamBinding? = null
+//    private val binding get() = _binding!!
+
+/*    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+
+        _binding = FragmentTeamBinding.inflate(inflater, container, false)
+        return binding.root
+
+    }*/
+}
Index: app/src/main/java/ru/practicum/android/diploma/ui/main/MainViewModel.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/ru/practicum/android/diploma/ui/main/MainViewModel.kt b/app/src/main/java/ru/practicum/android/diploma/ui/main/MainViewModel.kt
new file mode 100644
--- /dev/null	(date 1715338891074)
+++ b/app/src/main/java/ru/practicum/android/diploma/ui/main/MainViewModel.kt	(date 1715338891074)
@@ -0,0 +1,6 @@
+package ru.practicum.android.diploma.ui.main
+
+import androidx.lifecycle.ViewModel
+
+class MainViewModel: ViewModel() {
+}
Index: app/src/main/java/ru/practicum/android/diploma/ui/favorites/FavoritesViewModel.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/ru/practicum/android/diploma/ui/favorites/FavoritesViewModel.kt b/app/src/main/java/ru/practicum/android/diploma/ui/favorites/FavoritesViewModel.kt
new file mode 100644
--- /dev/null	(date 1715338891079)
+++ b/app/src/main/java/ru/practicum/android/diploma/ui/favorites/FavoritesViewModel.kt	(date 1715338891079)
@@ -0,0 +1,6 @@
+package ru.practicum.android.diploma.ui.favorites
+
+import androidx.lifecycle.ViewModel
+
+class FavoritesViewModel: ViewModel() {
+}
